app/about/page.tsx

```diff
@@ -1,35 +1,10 @@
-"use client";
-import { Box } from "@nypl/design-system-react-components";
-import aboutPageElements from "../data/aboutPageElements";
-import React from "react";
+import AboutPage from "app/components/aboutPage/aboutPage";
+import { Metadata } from "next";
```

Comment:
wait - I thought Metadata only worked on server components. Does this work with client components too or does this mean we're using server components now? (eyes emoji)

Reply:
read PR description <3

Reply:
I did and that's why I left a comment but I'm still confused! More surprised, actually. This feels like a big deal?

Reply:
I thought we couldn't use the DS with server components

Reply:
nvm, jumped the gun a bit. I see how this works now